### PR TITLE
mass-added the utf-8 marker

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Mass-added the marker to all .py files under `plugins` wihtout it.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME

plugins/lookup/nios_next_ip.py
plugins/lookup/nios_next_network.py
plugins/module_utils/_mount.py
plugins/module_utils/_netapp.py
plugins/module_utils/alicloud_ecs.py
plugins/module_utils/cloud.py
plugins/module_utils/database.py
plugins/module_utils/heroku.py
plugins/module_utils/hwc_utils.py
plugins/module_utils/ibm_sa_utils.py
plugins/module_utils/identity/keycloak/keycloak.py
plugins/module_utils/known_hosts.py
plugins/module_utils/linode.py
plugins/module_utils/manageiq.py
plugins/module_utils/memset.py
plugins/module_utils/net_tools/nios/api.py
plugins/module_utils/oneandone.py
plugins/module_utils/oneview.py
plugins/module_utils/online.py
plugins/module_utils/opennebula.py
plugins/module_utils/oracle/oci_utils.py
plugins/module_utils/rax.py
plugins/module_utils/redfish_utils.py
plugins/module_utils/redhat.py
plugins/module_utils/remote_management/lxca/common.py
plugins/module_utils/scaleway.py
plugins/module_utils/storage/emc/emc_vnx.py
plugins/module_utils/storage/hpe3par/hpe3par.py
plugins/module_utils/univention_umc.py
plugins/module_utils/utm_utils.py
plugins/modules/cloud/centurylink/clc_aa_policy.py
plugins/modules/cloud/centurylink/clc_alert_policy.py
plugins/modules/cloud/centurylink/clc_blueprint_package.py
plugins/modules/cloud/centurylink/clc_firewall_policy.py
plugins/modules/cloud/centurylink/clc_group.py
plugins/modules/cloud/centurylink/clc_loadbalancer.py
plugins/modules/cloud/centurylink/clc_modify_server.py
plugins/modules/cloud/centurylink/clc_publicip.py
plugins/modules/cloud/centurylink/clc_server.py
plugins/modules/cloud/centurylink/clc_server_snapshot.py
plugins/modules/cloud/heroku/heroku_collaborator.py
plugins/modules/cloud/linode/linode.py
plugins/modules/cloud/misc/proxmox.py
plugins/modules/cloud/misc/proxmox_template.py
plugins/modules/cloud/misc/xenserver_facts.py
plugins/modules/cloud/oneandone/oneandone_firewall_policy.py
plugins/modules/cloud/oneandone/oneandone_load_balancer.py
plugins/modules/cloud/oneandone/oneandone_monitoring_policy.py
plugins/modules/cloud/oneandone/oneandone_private_network.py
plugins/modules/cloud/oneandone/oneandone_public_ip.py
plugins/modules/cloud/oneandone/oneandone_server.py
plugins/modules/cloud/opennebula/one_host.py
plugins/modules/cloud/opennebula/one_template.py
plugins/modules/cloud/oracle/oci_vcn.py
plugins/modules/cloud/ovh/ovh_ip_failover.py
plugins/modules/cloud/ovh/ovh_ip_loadbalancing_backend.py
plugins/modules/cloud/packet/packet_device.py
plugins/modules/cloud/packet/packet_sshkey.py
plugins/modules/cloud/profitbricks/profitbricks.py
plugins/modules/cloud/profitbricks/profitbricks_datacenter.py
plugins/modules/cloud/profitbricks/profitbricks_nic.py
plugins/modules/cloud/profitbricks/profitbricks_volume.py
plugins/modules/cloud/profitbricks/profitbricks_volume_attachments.py
plugins/modules/cloud/pubnub/pubnub_blocks.py
plugins/modules/cloud/rackspace/rax.py
plugins/modules/cloud/rackspace/rax_cbs.py
plugins/modules/cloud/rackspace/rax_cbs_attachments.py
plugins/modules/cloud/rackspace/rax_cdb.py
plugins/modules/cloud/rackspace/rax_cdb_database.py
plugins/modules/cloud/rackspace/rax_cdb_user.py
plugins/modules/cloud/rackspace/rax_clb.py
plugins/modules/cloud/rackspace/rax_clb_nodes.py
plugins/modules/cloud/rackspace/rax_clb_ssl.py
plugins/modules/cloud/rackspace/rax_dns.py
plugins/modules/cloud/rackspace/rax_dns_record.py
plugins/modules/cloud/rackspace/rax_facts.py
plugins/modules/cloud/rackspace/rax_files.py
plugins/modules/cloud/rackspace/rax_files_objects.py
plugins/modules/cloud/rackspace/rax_identity.py
plugins/modules/cloud/rackspace/rax_keypair.py
plugins/modules/cloud/rackspace/rax_meta.py
plugins/modules/cloud/rackspace/rax_mon_alarm.py
plugins/modules/cloud/rackspace/rax_mon_check.py
plugins/modules/cloud/rackspace/rax_mon_entity.py
plugins/modules/cloud/rackspace/rax_mon_notification.py
plugins/modules/cloud/rackspace/rax_mon_notification_plan.py
plugins/modules/cloud/rackspace/rax_network.py
plugins/modules/cloud/rackspace/rax_queue.py
plugins/modules/cloud/rackspace/rax_scaling_group.py
plugins/modules/cloud/rackspace/rax_scaling_policy.py
plugins/modules/cloud/scaleway/scaleway_compute.py
plugins/modules/cloud/scaleway/scaleway_database_backup.py
plugins/modules/cloud/scaleway/scaleway_ip.py
plugins/modules/cloud/scaleway/scaleway_lb.py
plugins/modules/cloud/scaleway/scaleway_security_group.py
plugins/modules/cloud/scaleway/scaleway_security_group_rule.py
plugins/modules/cloud/scaleway/scaleway_sshkey.py
plugins/modules/cloud/scaleway/scaleway_user_data.py
plugins/modules/cloud/scaleway/scaleway_volume.py
plugins/modules/cloud/spotinst/spotinst_aws_elastigroup.py
plugins/modules/cloud/univention/udm_dns_record.py
plugins/modules/cloud/univention/udm_dns_zone.py
plugins/modules/cloud/univention/udm_group.py
plugins/modules/cloud/univention/udm_share.py
plugins/modules/cloud/univention/udm_user.py
plugins/modules/clustering/consul/consul.py
plugins/modules/clustering/consul/consul_acl.py
plugins/modules/clustering/consul/consul_kv.py
plugins/modules/clustering/etcd3.py
plugins/modules/clustering/znode.py
plugins/modules/database/aerospike/aerospike_migrations.py
plugins/modules/database/influxdb/influxdb_database.py
plugins/modules/database/influxdb/influxdb_retention_policy.py
plugins/modules/database/influxdb/influxdb_user.py
plugins/modules/files/sapcar_extract.py
plugins/modules/files/xattr.py
plugins/modules/monitoring/sensu/sensu_client.py
plugins/modules/monitoring/sensu/sensu_handler.py
plugins/modules/net_tools/dnsimple.py
plugins/modules/net_tools/ip_netns.py
plugins/modules/net_tools/ipinfoio_facts.py
plugins/modules/net_tools/ipwcli_dns.py
plugins/modules/net_tools/lldp.py
plugins/modules/net_tools/nios/nios_a_record.py
plugins/modules/net_tools/nios/nios_aaaa_record.py
plugins/modules/net_tools/nios/nios_cname_record.py
plugins/modules/net_tools/nios/nios_dns_view.py
plugins/modules/net_tools/nios/nios_fixed_address.py
plugins/modules/net_tools/nios/nios_host_record.py
plugins/modules/net_tools/nios/nios_member.py
plugins/modules/net_tools/nios/nios_mx_record.py
plugins/modules/net_tools/nios/nios_naptr_record.py
plugins/modules/net_tools/nios/nios_network.py
plugins/modules/net_tools/nios/nios_network_view.py
plugins/modules/net_tools/nios/nios_ptr_record.py
plugins/modules/net_tools/nios/nios_srv_record.py
plugins/modules/net_tools/nios/nios_txt_record.py
plugins/modules/net_tools/nios/nios_zone.py
plugins/modules/net_tools/nsupdate.py
plugins/modules/notification/syslogger.py
plugins/modules/packaging/language/pip_package_info.py
plugins/modules/packaging/os/redhat_subscription.py
plugins/modules/packaging/os/rhn_channel.py
plugins/modules/packaging/os/rhsm_release.py
plugins/modules/packaging/os/rhsm_repository.py
plugins/modules/packaging/os/rpm_ostree_pkg.py
plugins/modules/packaging/os/swupd.py
plugins/modules/remote_management/lxca/lxca_cmms.py
plugins/modules/remote_management/lxca/lxca_nodes.py
plugins/modules/remote_management/manageiq/manageiq_group.py
plugins/modules/remote_management/manageiq/manageiq_tenant.py
plugins/modules/remote_management/manageiq/manageiq_user.py
plugins/modules/remote_management/oneview/oneview_datacenter_info.py
plugins/modules/remote_management/oneview/oneview_enclosure_info.py
plugins/modules/remote_management/oneview/oneview_ethernet_network.py
plugins/modules/remote_management/oneview/oneview_ethernet_network_info.py
plugins/modules/remote_management/oneview/oneview_fc_network.py
plugins/modules/remote_management/oneview/oneview_fc_network_info.py
plugins/modules/remote_management/oneview/oneview_fcoe_network.py
plugins/modules/remote_management/oneview/oneview_fcoe_network_info.py
plugins/modules/remote_management/oneview/oneview_logical_interconnect_group.py
plugins/modules/remote_management/oneview/oneview_logical_interconnect_group_info.py
plugins/modules/remote_management/oneview/oneview_network_set.py
plugins/modules/remote_management/oneview/oneview_network_set_info.py
plugins/modules/remote_management/oneview/oneview_san_manager.py
plugins/modules/remote_management/oneview/oneview_san_manager_info.py
plugins/modules/source_control/github/github_issue.py
plugins/modules/source_control/github/github_key.py
plugins/modules/source_control/github/github_webhook.py
plugins/modules/source_control/github/github_webhook_info.py
plugins/modules/storage/emc/emc_vnx_sg_member.py
plugins/modules/storage/hpe3par/ss_3par_cpg.py
plugins/modules/system/lbu.py
plugins/modules/system/pids.py
plugins/modules/system/python_requirements_info.py
plugins/modules/system/selogin.py
plugins/modules/system/syspatch.py
plugins/modules/system/sysupgrade.py
plugins/modules/system/vdo.py
plugins/modules/web_infrastructure/jenkins_build.py
plugins/modules/web_infrastructure/jenkins_job.py
plugins/modules/web_infrastructure/jenkins_job_info.py
plugins/modules/web_infrastructure/sophos_utm/utm_aaa_group.py
plugins/modules/web_infrastructure/sophos_utm/utm_aaa_group_info.py
plugins/modules/web_infrastructure/sophos_utm/utm_ca_host_key_cert.py
plugins/modules/web_infrastructure/sophos_utm/utm_ca_host_key_cert_info.py
plugins/modules/web_infrastructure/sophos_utm/utm_dns_host.py
plugins/modules/web_infrastructure/sophos_utm/utm_network_interface_address.py
plugins/modules/web_infrastructure/sophos_utm/utm_network_interface_address_info.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_auth_profile.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_exception.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_frontend.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_frontend_info.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_location.py
plugins/modules/web_infrastructure/sophos_utm/utm_proxy_location_info.py

### CODE REVIEW

File: plugins/module_utils/univention_umc.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (felixfontein) [inline plugins/module_utils/univention_umc.py:2]: Now it's there twice.

**Review by felixfontein — COMMENTED** [review]
Everything else looks good.

**Comment by felixfontein** [comment]
@russoz btw there's no need to close/reopen a PR when tests fail. Simply restart the failing ones. You should be able to do that from GH's UI: open details for the "CI" job, and you'll see "Retry" links in the column on the left side for all failed runs. Click on one of these.

**Comment by russoz** [comment]
> @russoz btw there's no need to close/reopen a PR when tests fail. Simply restart the failing ones. You should be able to do that from GH's UI: open details for the "CI" job, and you'll see "Retry" links in the column on the left side for all failed runs. Click on one of these.

There were 40+ "failures", I wasn't going to retry one by one.

File: plugins/modules/cloud/univention/udm_dns_record.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (zanssa) [inline plugins/modules/cloud/univention/udm_dns_record.py:3]:
```suggestion

```

File: plugins/modules/cloud/univention/udm_dns_zone.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (zanssa) [inline plugins/modules/cloud/univention/udm_dns_zone.py:3]:
```suggestion

```

File: plugins/modules/cloud/univention/udm_group.py [hunk 4]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (zanssa) [inline plugins/modules/cloud/univention/udm_group.py:3]:
```suggestion

```

File: plugins/modules/cloud/univention/udm_share.py [hunk 5]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (zanssa) [inline plugins/modules/cloud/univention/udm_share.py:3]:
```suggestion

```

File: plugins/modules/cloud/univention/udm_user.py [hunk 6]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python
+# -*- coding: utf-8 -*-
 # -*- coding: UTF-8 -*-
```
Comment (zanssa) [inline plugins/modules/cloud/univention/udm_user.py:3]:
```suggestion

```

File: plugins/modules/cloud/univention/udm_dns_record.py (hunk 2 above)
Comment (russoz) [reply plugins/modules/cloud/univention/udm_dns_record.py]: Hi @zanssa , thanks for reviewing. @felixfontein had pointed that already, and I have just pushed the update.

This handful of files *had* the marker with `UTF-8` (uppercase), and I had missed that, hence assuming that all occurrences would ever be lowercase.

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
The following files do not have a proper marker:
plugins/modules/system/kernel_blacklist.py
plugins/modules/packaging/os/zypper_repository.py
plugins/modules/web_infrastructure/jenkins_plugin.py
plugins/modules/web_infrastructure/jenkins_script.py

**Comment by russoz** [comment]
> The following files do not have a proper marker:
> plugins/modules/system/kernel_blacklist.py
> plugins/modules/packaging/os/zypper_repository.py
> plugins/modules/web_infrastructure/jenkins_plugin.py
> plugins/modules/web_infrastructure/jenkins_script.py

Bummer, my bash-fu is no longer what it used to be

**Review by quidame — APPROVED** [review]
@russoz thanks for normalizing this !

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks a lot for doing this!
@zanssa @quidame thanks a lot for reviewing!

**Pull request merged**